src/python/pants/backend/codegen/protobuf/python/python_protobuf_subsystem.py

```diff
@@ -32,11 +34,24 @@ def register_options(cls, register):
                 "be automatically added to every `protobuf_library` target"
             ),
         )
+        register(
+            "--mypy-plugin",
```

Comment:
Open to other option names. I considered `--generate-type-stubs` as well.

Reply:
`--mypy-plugin` works!

Reply:
I think it's fine as-as.